compiler/ast/expressions.go

```diff
@@ -230,6 +235,28 @@ func (ie *IfExpression) String() string {
 	return out.String()
 }
 
+type ConditionalExpression struct {
```

Comment:
Oh, yes! I've never thought about it but now that you mention it, I think we can use this for case statement since ultimately ruby"s case statement is just a series of if, elsif and else statements! 😄 

Did you already support case statement? If not, I'm glad to work on it after finishing elsif 👍